apps/frontend-manage/src/components/evaluation/EvaluationUnavailableNotification.tsx

```diff
@@ -12,6 +29,68 @@ function EvaluationUnavailableNotification() {
         }}
         message={t('manage.evaluation.evaluationNotYetAvailable')}
       />
+
+      <div className="mt-4 flex flex-row gap-8 text-xs text-gray-500">
```

Comment:
Could we maybe stick to at least small text here? text-xs seems very very small and is barely readable

```suggestion
      <div className="mt-4 flex flex-row gap-8 text-sm text-gray-500">
```